news/2021-11-08-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,132 @@
+
+### New Flashlight difficulty skill
+
+Flashlight in difficulty calculation is now revamped, as it moves from an object count-based multiplier on the Aim skill to a brand new skill of its own. 
+
+The difficulty that Flashlight adds to a map is now measured much more accurately, thanks to [**MBmasher**](https://osu.ppy.sh/users/4498616)’s change in [this pull request](https://github.com/ppy/osu/pull/14217).
+
+Previously, the object count-based method posed a number of issues. Flow aim (measured when playing streams) is mostly represented by the speed skill, not aim. Additionally, the aim skill (surprisingly) only measures the aim difficulty, not the flashlight difficulty. To put this simply, there will be types of jumps that are hard to aim and quite easy to read with flashlight, and vice versa.
+
+For example, let’s consider a familiar type of jump pattern: a back and forth. This is an example of a jump that can have high aim difficulty, but relatively low flashlight difficulty. The distance between the first object and the third object in a back and forth is typically small enough for it to be within the flashlight radius, thus allowing a player to read the position of that circle very easily.
+
+![](/wiki/shared/news/2021-11-08-performance-points-star-rating-updates/visible-fl-area.png)
+
+The creation of a Flashlight skill makes it possible to have it contribute to Star Rating. [This has been enabled](https://github.com/ppy/osu/pull/14753) following the results of the osu! performance points and star rating survey, which revealed that players desire star rating to be a metric of difficulty, and that they want subjective skills included. Ask and thou shall receive!
+
+It is also worth noting that the Hidden mod applies a 1.3x multiplier on the Flashlight skill. The visibility of hitobjects are decreased significantly due to fading of circles and the absence of approach circles.
+
+Find yourself curious about the details? [MBmasher](https://osu.ppy.sh/users/4498616)’s write up about the implementation will very likely scratch that itch, so feel free to [**indulge in some Flashlight theory**](https://docs.google.com/document/d/11FBEEPXK8yI2d5uTZuebJC3S9nVUKgMD8WdvblERyYA/edit).
+
+### Speed hard cap removal
+
+The speed hard cap in osu! has been removed by [**emu1337**](https://osu.ppy.sh/users/2185987) and [**Apo11o**](https://osu.ppy.sh/users/9558549) in [this pull request](https://github.com/ppy/osu/pull/14617), allowing 300+ BPM streams to be weighed accurately. 
+
+The speed cap was previously set to 300 BPM to prevent abuse cases, which means that calculating higher BPMs requires consideration into those cases. Two factors are implemented with regard to this:
```

Comment:
kinda reads weird. maybe this works better?

```suggestion
The speed cap was previously set to 300 BPM to prevent abuse cases, which means that calculating difficulty on beatmaps with higher BPM requires special consideration. Two adjustments were made to account for that:
```